src/linear_algebra/bilinear_form.lean

```diff
@@ -948,14 +948,18 @@ end to_matrix
 
 end matrix
 
-namespace refl_bilin_form
-
-open refl_bilin_form bilin_form
+namespace bilin_form
 
 /-- The proposition that a bilinear form is reflexive -/
 def is_refl (B : bilin_form R M) : Prop := ∀ (x y : M), B x y = 0 → B y x = 0
 
-variable (H : is_refl B)
+end bilin_form
+
+namespace refl_bilin_form
```

Comment:
I think this namespace should change to `bilin_form.is_refl`, and same for the alt / symm namespaces

Reply:
there are several instances of the `is_refl` Lemma, what should we do about these? Rename to `symm_is_refl` and `alt_is_refl`?

Reply:
They'll be in different namespaces still; the full names should be `bilin_form.is_symm.is_refl` and `bilin_form.is_alt.is_refl`